planex/makesrpm.py

```diff
@@ -105,37 +88,54 @@ def main(argv):
     """
 
     args = parse_args_or_exit(argv)
-    tmp_dirpath, tmp_specs, tmp_sources = setup_tmp_area()
-    tmp_specfile = os.path.join(tmp_specs, os.path.basename(args.spec))
+    tmpdir = tempfile.mkdtemp()
+    tmp_specfile = os.path.join(tmpdir, os.path.basename(args.spec))
+
+    link = None
+    if args.link:
+        link = Link(args.link)
 
     try:
-        # Copy files to temporary working area
-        copyfile(args.spec, tmp_specfile)
-        tarball_filters = ['.tar.gz', '.tar.bz2']
+        # Copy spec to working area
+        shutil.copyfile(args.spec, tmp_specfile)
 
+        # Copy sources to working area, rewriting spec as needed
+        tarball_filters = ['.tar.gz', '.tar.bz2']
```

Comment:
This has always been the case in planex-make-srpm.   I'm not going to change it in this pull request.